### PR TITLE
Include a solr cloud health check before index swap instructions

### PR DESCRIPTION
**Pull request by sandbergja**

### CODE REVIEW

File: docs/indexing.md [hunk 1]
```diff
@@ -197,7 +197,15 @@ above to do this)
 
 ### Swap in the new index
 
-First, update the index managers to have the new solr_collection values.
+Before you swap in the new index, make sure that the solr cloud is all working as expected:
+
+1. Open the solr console
+2. Open the Cloud sub-menu
```
Comment (carolyncole) [inline docs/indexing.md:203]: In Markdown this should just be `1.` for all the numbers and then markdown will take care of the numbering.

File: docs/indexing.md [hunk 2]
```diff
@@ -197,7 +197,15 @@ above to do this)
 
 ### Swap in the new index
 
-First, update the index managers to have the new solr_collection values.
+Before you swap in the new index, make sure that the solr cloud is all working as expected:
+
+1. Open the solr console
```
Comment (carolyncole) [inline docs/indexing.md:202]: We should link to the connecting to the solr cloud portion of the doc `[Open the solr console](#accessing-the-solr-admin-ui)` I believe is the correct link.

**Review by carolyncole — CHANGES_REQUESTED** [review]
Nice addition!  Just a couple of comments.

File: docs/indexing.md [hunk 3]
```diff
@@ -50,6 +50,16 @@ You can select a collection and use the "query" menu option to check how many do
 
 ## Creating a Full Index
 
+### Check solr cloud health
```
Comment (carolyncole) [inline docs/indexing.md:53]: @sandbergja can we now just link to this section in the bottom?

Comment (christinach) [reply docs/indexing.md:53]: @sandbergja it looks great. Can we move Check solr cloud health before the swap section https://github.com/pulibrary/bibdata/blob/d3d64ec800cb3b814a77b952dfe957db6cf5025e/docs/indexing.md#swap-in-the-new-index

**Review by carolyncole — APPROVED** [review]

**Pull request merged**